doc/release-notes.md

```diff
@@ -62,6 +62,10 @@ RPC changes
 ### Low-level changes
 
 - The `fundrawtransaction` rpc will reject the previously deprecated `reserveChangeKey` option.
+- New `fees` field introduced in `getrawmempool`, `getmempoolancestors`, `getmempooldescendants` and
+   `getmempoolentry` when verbosity is set to `true` with sub-fields `ancestor`, `base`, `modified`
```

Comment:
Not sure what ```when verbosity is set to `true` ``` means.

Reply:
how about if I change the wording to "(when verbosity=true argument is supplied)"?